src/main/kotlin/no/nav/familie/ba/sak/kjerne/vedtak/vedtaksperiode/VedtaksperiodeMedBegrunnelserController.kt

```diff
@@ -134,6 +135,15 @@ class VedtaksperiodeMedBegrunnelserController(
         return ResponseEntity.ok(Ressurs.Companion.success(begrunnelser.toSet()))
     }
 
+    @GetMapping(path = ["/behandling/{behandlingId}/hent-vedtaksperioder"])
+    fun hentPersonerMedUgyldigEtterbetalingsperiode(
```

Comment:
Burde denne metodenavnet endres? Nå har den ikke så mye med endepunkt navnet å gjøre.
```suggestion
    fun hentRestUtvidetVedtaksperiodeMedBegrunnelser(
```

Reply:
Bra spotta! Fikset i [Fiks navn på funksjon](https://github.com/navikt/familie-ba-sak/pull/3869/commits/b338703b8c738d5e90c2b7a2d203d2dc85571b11)